tests/Amadeus/Client/Struct/Fare/MasterPricerCalendarTest.php

```diff
@@ -151,8 +152,9 @@ public function testCanMakeRequestWithTripDetails()
         $this->assertEquals('KBP', $message->itinerary[0]->departureLocalization->departurePoint->locationId);
         $this->assertEquals('JFK', $message->itinerary[0]->arrivalLocalization->arrivalPointDetails->locationId);
 
+        $this->assertEquals('161118', $message->itinerary[0]->timeDetails->firstDateTimeDetail->date);
+        $this->assertEquals(3, $message->itinerary[0]->timeDetails->rangeOfDate->dayInterval);
+        $this->assertEquals(RangeOfDate::RANGEMODE_MINUS_PLUS, $message->itinerary[0]->timeDetails->rangeOfDate->rangeQualifier);
         $this->assertEquals(MPTripDetails::FLEXIBILITY_COMBINED, $message->itinerary[0]->timeDetails->tripDetails->flexibilityQualifier);
```

Comment:
When asserting if the message structure is as expected you should use the `Struct` files and not the `RequestOptions`. The struct files may change without the request options having been modified (or the other way round)

Reply:
Yeah... It makes sense